### ISSUE TITLE
Having trouble with device actions

### ISSUE THREAD

**Issue by simplejeremy** [androidthings/sample-googleassistant#80]
I'm trying to test the capability of controlling hardware (e.g. LEDs) on the board using Google Voice Assistant. When I give the commands "Turn on" and "Turn off" to control the LED using the NXP i.MX7D Starter Kit, I get "Sorry, power control is not yet supported."

The device has been registered using the registration UI and DEVICE_MODEL_ID was modified to match the device model ID from the webpage. I didn't modify DEVICE_INSTANCE_ID so the value is initialized to "PLACEHOLDER".

**Comment by Fleker** [comment]
Are the credentials on the device the same account as the device model you registered in the console?

**Comment by simplejeremy** [comment]
Yes, it uses the same credentials with the JSON file downloaded from my account.

**Comment by Fleker** [comment]
Have you enabled the _OnOff_ trait for that device model id in the console?

Can you register a device with that device model id, and update the `DEVICE_INSTANCE_ID` constant?

**Comment by simplejeremy** [comment]
Yes, it's selected under "Supported traits" on the Linked device models webpage.

When I try to follow the instructions on [this webpage](https://developers.google.com/assistant/sdk/reference/device-registration/register-device-manual) to get DEVICE_INSTANCE_ID, I get stuck at where $ACCESS_TOKEN is written since my sample-googleassistant\app\src\main\res\raw\credentials.json doesn't contain the property with the key "token" in the string.

**Comment by Fleker** [comment]
If you re-authenticate the tool, then you shouldn't need to save it to the credentials.json
